public/audioSearchApi.html

```diff
@@ -237,7 +237,8 @@ <h4>{{album}}</h4>
             source: 'custom_embed',
             action,
             data,
-            key: parseQueryString()['k']
+            key: parseQueryString()['k'],
+            isAnsRequired: false,
```

Comment:
remove this too

Reply:
Just to be sure, do you mean to remove `isAnsRequired: false,`?

Reply:
yes